src/libsync/theme.cpp

```diff
@@ -91,6 +91,11 @@ QString Theme::statusHeaderText(SyncResult::Status status) const
     return resultStr;
 }
 
+bool Theme::isBranded() const
+{
+    return appNameGUI() != QStringLiteral("Nextcloud");
```

Comment:
Just a comment, no request: Would it make sense to just add another few lines (one const return func in theme.h/cpp isBranded, returning APPLICATION_BRANDED and set(APPLICATION_BRANDED... in Application.cmake)? I am obviously totally unaware of how branding is done but intuitively I think it could possible that branding does not **have to** change the application name?

Reply:
Yes, longer term we should do it differently.